### PR TITLE
DDP 4094 Add "required" to Physician Institution Components

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadinstitute.atlassian.net/browse/DDP-4094
Changes to support "required" for physician institution components
Add required for MBC blood release Physician component

### CODE REVIEW

**Comment by ssettipalli** [comment]
please run tests

File: pepper-apis/src/main/resources/db-changes/schema/DDP-4094-add-required-to-institution-physician-component.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet id="20200206-add-required-to-institution-physician-component" author="sampath">
+
+        <addColumn tableName="institution_physician_component">
+            <column name="required" type="boolean" defaultValueBoolean="false">
```
Comment (yufengwng) [inline pepper-apis/src/main/resources/db-changes/schema/DDP-4094-add-required-to-institution-physician-component.xml:10]: Should we make this non-null?

**Review by yufengwng — COMMENTED** [review]
Looks like there's an API addition in here. Can you update the OpenAPI docs as well?

**Comment by ssettipalli** [comment]
> Looks like there's an API addition in here. Can you update the OpenAPI docs as well?

good find. Thank you. updated API docs for Physician and Institution components

**Comment by ssettipalli** [comment]
please run tests

**Review by yufengwng — DISMISSED** [review]
LGTM

**Comment by ssettipalli** [comment]
please run tests

**Review by raiks2 — APPROVED** [review]

**Pull request merged**